ci/e2e.azure-pipelines.yml

```
@@ -0,0 +1,38 @@
+trigger: none
```

I think this is not sufficient to make this pipeline only run manually: we still see automated runs of the pipeline in GitLab ("PR automated for #22"). GitLab appears to differentiate continuous integration on individual commits, and CI checks for a PR.

```suggestion
trigger: none # Disable individual commit runs. https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/azure-repos-git?view=azure-devops&tabs=yaml#opting-out-of-ci
pr: none # Disable PR runs. https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/azure-repos-git?view=azure-devops&tabs=yaml#pr-triggers
```